### PR TITLE
cleanup(angular): add unit tests coverage for generating a minimal application

### PR DESCRIPTION
**Pull request by leosvelperez**
## Current Behavior

Generating a minimal application is not covered by unit tests. There were issues with the generation that could have been caught by them.

## Expected Behavior

Generating a minimal application is covered by unit tests.

## Related Issue(s)

Fixes

### CODE REVIEW

**Comment by leosvelperez** [comment]
Holding this off until #15698 is merged. That PR is a bigger change to our application generation and might end up addressing the issue this one is resolving.

**Review by Coly010 — APPROVED** [review]

**Pull request merged**
